DigitalLearningSolutions.Web/Views/Shared/Components/NumberOfAdministrators/Default.cshtml

```diff
@@ -0,0 +1,78 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common
+@model NumberOfAdministratorsViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/numberOfAdministratorsViewComponent.css")" asp-append-version="true">
+
+@{
+  var trainerDisplay = Model.TrainerSpotsAvailable == -1 ? Model.Trainers.ToString() : Model.Trainers + "/" + Model.TrainerSpotsAvailable;
+  var cmsAdministratorsDisplay = Model.CmsAdministratorSpotsAvailable == -1 ? Model.CmsAdministrators.ToString() : Model.CmsAdministrators + "/" + Model.CmsAdministratorSpotsAvailable;
+  var cmsManagersDisplay = Model.CmsManagerSpotsAvailable == -1 ? Model.CmsManagers.ToString() : Model.CmsManagers + "/" + Model.CmsManagerSpotsAvailable;
+  var ccLicencesDisplay = Model.CcLicencesAvailable == -1 ? Model.CcLicences.ToString() : Model.CcLicences + "/" + Model.CcLicencesAvailable;

```

Comment:
Looks good, but maybe worth putting this logic in the view model so we can write some simple unit tests for this?